lib/client-api/src/story_store.ts

```diff
@@ -283,15 +283,16 @@ export default class StoryStore {
       storyParameters
     );
 
-    const parameters = this._parameterEnhancers.reduce(
+    const parameters = this._argTypesEnhancers.reduce(
       (accumlatedParameters, enhancer) => ({
         ...accumlatedParameters,
-        ...enhancer({
-          ...identification,
-          parameters: accumlatedParameters,
-          args: {},
-          globalArgs: {},
-        }),
+        argTypes:
+          enhancer({
+            ...identification,
+            parameters: accumlatedParameters,
+            args: {},
+            globalArgs: {},
+          }) || accumlatedParameters.argTypes,
```

Comment:
So an `ArgTypesEnhancer` can return falsey? Are you sure we shouldn't just make them return `context.parameters.argTypes`?

Reply:
If not we should add a test for this.